src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -254,8 +263,6 @@ function plant_absorbed_pfd(
     frac_diff::FT,
 ) where {FT}
 
-    (; G_Function, Ω, n_layers) = RT.parameters
-
     # Get the current leaf angular distribution value based on the solar zenith angle
     G = compute_G(G_Function, θs)
```

Comment:
To handle this, we would need to make a field `p.canopy.radiative_transfer.G`, and update it prior to calling `plant_absorbed_pfd`. then we could pass in the field and broadcast over it. The update would dispatch off of the type of Gfunction. This could be done in another PR or here. happy to discuss more!

Reply:
Is G_function something that is space varying? If not, why would we need to make the field p.canopy.radiative_transfer.G?

Reply:
The G function has spatially varying parameters. The function is always the same function of those parameters and the independent variable theta_s. So the dependent variable (which we call G) is spatially varying. we can store that, not the function

Reply:
I thought there are two different G functions: ConstantGFunction and CLMGFunction.

If we do not have access to theta_s at the time of constructing the Radiative transfer model parameters struct, then don't we still need to store the function type or at least its parameters, so G can be calculated when theta_s is known?